client/src/i18n/locales/pl.json

```diff
@@ -38,15 +38,17 @@
     "AboutMistakesJournal": {
       "HeaderFirstPart": "Wykorzystaj w pełni swoje błędy",
       "HeaderSecondPart": "prowadząc",
+      "EarlyAccessHeader": "Witamy we wczesnym dostępie!",
+      "EarlyAccessText": "Przekonaj się sam i daj znać do myślisz!",
       "Step1": {
         "StepName": "Rozpoznaj",
         "Header": "Błędy są cenne",
-        "Text": "Pomożemy Ci zapisywać oraz organizować Twoje błędy, tak aby czerpać z nich jak najwięcej."
+        "Text": "Pomożemy Ci zapisywać oraz organizować Twoje błędy, tak abyś mógł z nich czerpać jak najwięcej."
       },
       "Step2": {
         "StepName": "Analizuj",
         "Header": "Znajdź rozwiązania",
-        "Text": "Niektóre błędy nie są proste, ale zrobimy co w naszej mocy, aby pomóc Ci zreflektować się nad nimi, tak abyś miał porady na przyszłość."
+        "Text": "Niektóre błędy nie są proste ale zrobimy co w naszej mocy, aby pomóc Ci zreflektować się nad nimi, tak abyś miał porady na przyszłość."
```

Comment:
Why removing `,`? I suppose in polish you use `,` before `ale`.

Reply:
Szymons translation